.github/buildomat/phd-run-with-args.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Unpacks and executes the PHD runner in the Buildomat environment, passing
+# through to the runner any arguments that were passed to this script.
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+indir="/input"
+indir_suffix="phd-build/out/*.tar.gz"
+phddir="$PWD/phd-test"
+
+# Put artifacts on the runner's SSDs (the /work ramdisk is small by design, too
+# small for images of any appreciable size).
+pfexec zpool create -f phd-artifacts c1t1d0 c2t1d0
+artifactdir="/phd-artifacts"
+
+banner 'Inputs'
+find $indir -ls
+
+rm -rf "$phddir"
+mkdir "$phddir"
+
+for p in $indir/$indir_suffix; do
+	tar xzvf $p -C $phddir
+	for f in $(tar tf "$p"); do
+		chmod +x "$phddir/$f"
+	done
+done
+
+ls $phddir
+
+banner 'Setup'
+tmpdir="/tmp/propolis-phd"
+if [ ! -d "$tmpdir" ]; then
+	mkdir $tmpdir
+fi
+
+banner 'Tests'
+
+runner="$phddir/phd-runner"
+artifacts="$phddir/artifacts.toml"
+propolis="$phddir/propolis-server"
+
+ls $runner
+ls $artifacts
+ls $propolis
+
+args=()
+args+=($runner '--emit-bunyan' 'run')
+args+=('--propolis-server-cmd' $propolis)
+args+=('--base-propolis-branch' 'master')
```

Comment:
As I mentioned above, I think it might make more sense to have only the migration-from-base job pass this argument, so that we don't have to download a base Propolis artifact when we're not running the tests that need it.